src/decorators/types.ts

```diff
@@ -17,6 +17,7 @@ export type SubscriptionFilterFunc = (
 
 export interface DecoratorTypeOptions {
   nullable?: boolean;
+  complexity?: number;
```

Comment:
`graphql-query-complexity` allows to calculate it dynamically: `complexity: (args, childComplexity) => childComplexity * args.count,`